### DISCUSSION TITLE
Split "Hide the activity notification badge" between Dashboard and Activity

### DISCUSSION THREAD

**Discussion by laurapigeon** [AprilSylph/XKit-Rewritten#1132] [Suggestion Box]
I'd like the option to remove the notification badge for the dashboard only, to have a badge only when interacted with, as opposed to any time anyone I follow posts anything, without having to unfollow everyone.

**Comment by marcustyphoon** [comment]
You're talking about the badge on the "home" button (that looks like a house) which appears whenever there are new posts to view on your timeline, right?

**Reply by laurapigeon** [reply]
Yeah, separate from the lightning bolt interaction-based one.

**Reply by marcustyphoon** [reply]
Gotcha! I would also love a tweak that does this (hides the "dashboard" / "home" notification badge).
